packages/discord.js/typings/index.d.ts

```diff
@@ -5270,7 +5270,7 @@ export interface GuildMembersChunk {
   nonce: string | undefined;
 }
 
-type NonPartialGroupDMChannel<Structure extends { channel: Channel }> = Structure & {
+export type NonPartialGroupDMChannel<Structure extends { channel: Channel }> = Structure & {
```

Comment:
If we want to expose this lets give this a better name. As it stands, this name is very... confuse